app/assets/javascripts/discourse/app/components/d-lightbox/body.hbs

```diff
@@ -0,0 +1,57 @@
+<div
+  class="d-lightbox__body"
+  tabindex="-1"
+  {{on "touchstart" @onTouchstart passive=true capture=true}}
+  {{on "touchend" @onTouchend passive=true capture=true}}
+  {{on "click" @toggleZoom passive=true}}
+>
+  {{#if @shouldDisplayMainImageArrows}}
+    <DLightbox::Button
```

Comment:
Could you justify why we need these custom buttons please? It would be better to rely on our own DButton and improve them if necessary